cypress/support/getMarkdownRenderer.ts

```diff
@@ -13,16 +15,16 @@ declare namespace Cypress {
 }
 
 Cypress.Commands.add('getMarkdownRenderer', () => {
```

Comment:
```suggestion
Cypress.Commands.add('getMarkdownRenderer', (): Chainable<JQuery<HTMLElement>> => {
```

Reply:
Reverted the type because typescript freaked out.